### ISSUE TITLE
Brandless version.

### ISSUE THREAD

**Issue by theraot** [fawazahmed0/the-solo-developers-manifesto#3]
This is a suggestion:

Create a version of the manifesto without any mention of a software, service, company or brand. Create a separate document with recommendations for software and services to use.

---

Motivation:

Software and services, in particular privative ones, could fall out of support. The companies that provide them could discontinue them. Or the companies themselves could cease to exist. As a result, it would be impossible to honor this manifesto.

There are other reasons why somebody could be in a situation where the particular software and service could prevent them to honor the manifesto. 

---

For example, the manifesto says to use MS Paint. There was a point where Microsoft said they were to discontinue MS Paint, and - arguably - thanks to people rising their voice that didn't happen. However, they can still do it. If the software were libre and open source, that would not be an issue. If the developer of such software were to stop working on it, the source code code be legally taken by others to continue its development.

You also suggest online software. Which also share the problem above, except it is worse, if the company behind it ends the service, you could not use an old copy. Plus, online software require access to the Internet, which is not guaranteed.

Legal access to MS Paint is contingent on a paid Windows license, which not everybody might have access to. Signing this manifesto would mean agreeing to pay a Windows license (or alternatively break copyright lays), among other things. If the software were actually free that would be a problem. The issue with Windows licenses would extend to any other software that only runs on Windows. Thus, the selection of software would have to be portable too.

In lieu of figuring out software that anybody could use, regardless of platform, license, cost, or access to the Internet, I suggest to leave out exactly what software out of the manifesto, and let everybody figure out what to use, depending on their situation. If you still want an opinionated list of software recommendation, I suggest to keep it in a separate document.

---

This is, of course, just a suggestion. The same way you can choose to not do it, I can choose to not agree with your manifesto.

**Comment by fawazahmed0** [comment]
> Create a version of the manifesto without any mention of a software, service, company or brand. Create a separate document with recommendations for software and services to use.

I agree that I was pretty biased when I wrote this manifesto, for example being a JavaScript dev myself, I listed [StandardJS](https://standardjs.com/) which is for JavaScript only and there are many more examples of it.

I am ok, if you create a PR and add other tools in this manifesto

> Software and services, in particular privative ones, could fall out of support.
I will try to keep the tools list updated in the manifesto, and would also love to have you add the productivity tools that is used by your programming language community.

> . If you still want an opinionated list of software recommendation, I suggest to keep it in a separate document.
I think keeping the software suggestions in the manifesto will make it easier for people to implement the manifesto.

And I really would like to thank you for writing your suggestion, I will keep this issue open & will look into it in future when the software recommendation list starts to clutter up in the manifesto, I think that will be the best time to make a separate doc for each programming language community
